llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -8998,6 +9108,16 @@ static SetVector<VPIRInstruction *> collectUsersInExitBlocks(
           if (ExitVPBB->getSinglePredecessor() == MiddleVPBB)
             continue;
         }
+        // Exit values for CSAs are computed and updated outside of VPlan and
```

Comment:
I am missing where the exit values are computed outside of VPlan. What's preventing them from being competed in VPlan?